coda/coda_mdstore/presentation.py

```diff
@@ -32,7 +31,7 @@ def getFileList(url):
     """
 
     fileList = []
-    handle = urllib2.urlopen(url)
+    handle = urllib.request.urlopen(url)
     soup = BSoup(handle)
     trList = soup.findAll('tr')
```

Comment:
So this has me a bit worried about our tests. The `findAll` method of BeautifulSoup 3 was changed in bs4 to be `find_all`. There are other changes as well that may apply to our code. We need to be careful to check for changes like that when upgrading dependencies beyond just a patch or minor version. For bs4, you can see a helpful guide [here](https://www.crummy.com/software/BeautifulSoup/bs4/doc/#porting-code-to-bs4), but you may want to take this time to review the other dependency upgrades too to see if they require other modifications.

Reply:
I got curious as to whether it really _was_ working (contrary to the documentation) or if our tests weren't catching it. It seems we got the better result of it just being a failsafe in bs4 for just this kind of thing. So, the `findAll` still works, as it just points to the same `find_all` method as you can see [here](https://bazaar.launchpad.net/~leonardr/beautifulsoup/bs4/view/head:/bs4/element.py#L1303), but still, we don't have a guarantee on how long that will last. We should use the specified API for the version that we're using.

Reply:
I don't see tests testing that piece of code using `findAll` which is why tests didn't fail. I'll see and add tests for such pieces of code.